README.md

```diff
@@ -99,13 +99,3 @@ A number of plugins that originally resided in `backstage/backstage` monorepo ha
 - `vault`
 - `xcmetrics`
 </details>
-
-### Migration process
-
-The migration of plugins from the `backstage/backstage` monorepo to the `community-plugins` repository was automated under the `community-cli` tool.
```

Comment:
Is it worth to move this information to CONTRIBUTING.md, or to also remove the [associated script](https://github.com/backstage/community-plugins/blob/main/workspaces/repo-tools/packages/cli/src/commands/plugin/migrate.ts)?

Removing this information from the README appears to leave the `@backstage-community/cli migrate` largely undocumented

Reply:
I think the script is okay to be less documented for now as it's not ready or intended for general consumption in its current form (as @04kash mentioned). But, equally, I don't think we want to jump to removing the actual tooling/script while there are plans to improve it.